### PR TITLE
initializedNewTables

### PR DESCRIPTION
**Pull request by elmgren**
Added the tables DATASET, DISTRIBUTION_CHANNEL, SYSTEM and som junctiontables for these.

Deleted old V1.1 migration file and added everything in the first file.

### CODE REVIEW

File: data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql [hunk 1]
```diff
@@ -1,8 +1,6 @@
 CREATE SCHEMA IF NOT EXISTS BACKEND_SCHEMA;
 
 CREATE SEQUENCE IF NOT EXISTS BACKEND_SCHEMA.SEQ_INFORMATION_TYPE;
-CREATE SEQUENCE IF NOT EXISTS BACKEND_SCHEMA.SEQ_CODELIST;
```
Comment (skotsj) [inline data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql]: Skal denne bort?

File: data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql [hunk 2]
```diff
@@ -32,3 +30,70 @@ CREATE TABLE IF NOT EXISTS BACKEND_SCHEMA.INFORMATION_TYPE
...
+    LAST_MODIFIED_DATE   TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS BACKEND_SCHEMA.DISTRIBUTION_CHANNEL
+(
+    DISTRIBUTION_CHANNEL_ID UUID PRIMARY KEY,
+    NAME                    VARCHAR(100) UNIQUE NOT NULL,
+    DESCRIPTION             TEXT                NOT NULL,
+    PRODUCERS               TEXT                NOT NULL,
+    CONSUMERS               TEXT                NOT NULL,
```
Comment (skotsj) [inline data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql]: PRODUCERS               TEXT                NOT NULL,
    CONSUMERS               TEXT                NOT NULL,

Skal vel bort

Comment (elmgren) [reply data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql]: Fixed

File: data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql (hunk 1 above)
Comment (elmgren) [reply data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql]: Ja den blir ikke brukt i codelistobjektet.

**Pull request merged**